client/src/Components/App.jsx

```diff
@@ -1,18 +1,128 @@
 import React from 'react';
+import axios from 'axios';
+import isPosOrNeg from '../utils/utilities.js';
+import Reviews from './Reviews.jsx';
+import style from '../Styles/App.css';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      reviews: [],
+      overallPosOrNeg: '',
+      recentPosOrNeg: '',
+      recent: []
+    };
+  }
+
+  componentDidMount () {
+    axios.get('/api/reviews/' + 'Stardew Valley')
+    .then((data) => {
+      this.setState({
+        reviews: data.data
+      });
+    })
+    .then(() => {
+      let posOrNeg = isPosOrNeg(this.state.reviews);
+      this.setState({
+        overallPosOrNeg: posOrNeg
+      });
+    })
+    .then(() => {
+      let recent = [];
+      let today = new Date();
+      let thirtyDaysAgo = new Date(today - (30 * 86400000));
+      for (let i = 0; i < this.state.reviews.length; ++i) {
```

Comment:
did you do ++i because you intentionally wanted to skip index 0?

Reply:
++i is the same as i++, but it just adds faster? Index 0 isn't skipped, it's just that the increment happens faster, from what I understand. Udacity's beginner javascript tutorial explains the difference.